### PR TITLE
HARVESTER: polish namespace

### PR DESCRIPTION
**Pull request by WuJun2016**
#### PR Checklist
- Is this a multi-tenancy feature/bug?
    - [x] Yes, the relevant RBAC changes are at:
- Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?
    ~- [ ] Yes, the relevant PR is at:~
- Are backend engineers aware of UI changes?
    ~- [ ] Yes, the backend owner is:~

Fixes #

### Screenshot/Video

- single cluster
image

image

- multi cluster
image

image

image

### CODE REVIEW

File: pkg/harvester/store/harvester-store/getters.ts [hunk 1]
```diff
@@ -1 +1,127 @@
...
+
+      if (orphans.length) {
+        if (!firstProject) {
+          divider(out);
+        }
+
+        out.push({
+          id:    ALL_ORPHANS,
+          kind:  'project',
+          label: 'Not in a Project',
```
Comment (n313893254) [inline pkg/harvester/store/harvester-store/getters.ts:93]:
```suggestion
          label: rootGetters['i18n/t']('nav.ns.orphan'),
```

**Review by n313893254 — CHANGES_REQUESTED** [review]
Others LGTM

File: pkg/harvester/store/harvester-store/getters.ts (hunk 1 above)
Comment (WuJun2016) [reply pkg/harvester/store/harvester-store/getters.ts]: Done

File: pkg/harvester/models/namespace.js [hunk 2]
```diff
@@ -33,4 +51,34 @@ export default class HciNamespace extends namespace {
...
+      return true;
+    }
+
+    const isSettingSystemNamespace = this.$rootGetters['systemNamespaces'].includes(this.metadata.name);
+
+    if (isSettingSystemNamespace) {
+      return true;
+    }
+
+    return !!OBSCURE_NAMESPACE_PREFIX.some(prefix => this.metadata.name.startsWith(prefix));
```
[the remark this answers is not in the document]
Comment (WuJun2016) [reply pkg/harvester/models/namespace.js]: Done

**Comment by WuJun2016** [comment]
Done, PTAL again, thanks

**Review by n313893254 — APPROVED** [review]
LGTM

**Pull request merged**